common/dropwizard/src/test/java/com/bazaarvoice/emodb/common/dropwizard/log/DefaultRateLimitedLogFactoryTest.java

```diff
@@ -1,48 +1,43 @@
-package com.bazaarvoice.emodb.databus.core;
+package com.bazaarvoice.emodb.common.dropwizard.log;
 
 import io.dropwizard.util.Duration;
 import org.mockito.ArgumentCaptor;
 import org.mockito.Matchers;
+import org.mockito.Mockito;
 import org.slf4j.Logger;
+import org.testng.Assert;
 import org.testng.annotations.Test;
 
 import java.util.concurrent.ScheduledExecutorService;
 import java.util.concurrent.TimeUnit;
 
-import static org.mockito.Matchers.eq;
-import static org.mockito.Mockito.mock;
-import static org.mockito.Mockito.reset;
-import static org.mockito.Mockito.verify;
-import static org.mockito.Mockito.verifyNoMoreInteractions;
-import static org.testng.Assert.assertEquals;
```

Comment:
I moved the DefaultRateLimitedLog and its counterparts to common, and while doing to realized that all of these imports were unused. I figured that since git was going to track the whole file during the move, I might as well remove them.